lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
```

Comment:
I completely agree with you - the presence of a `Sender` module next to `Mailer` is confusing and feels redundant. Also, I kind-of have the same opinion about `HexWeb.Mail` (the email "builder")  v.s. `HexWeb.Email` (the model). WDYT?